packages/ingest/granule.js

```diff
@@ -66,7 +66,8 @@ class Discover {
     this.collection.files.forEach((f) => {
       this.regexes[f.regex] = {
         collection: this.collection.name,
-        bucket: this.buckets[f.bucket].name
+        bucket: this.buckets[f.bucket].name,
```

Comment:
Changes to this file support adding the Collection fileType object on granule discovery (e.g. DiscoverGranules) - this allows fileTypes to be populated for that ingest path